pkgs/by-name/ba/baballonia/package.nix

```
@@ -0,0 +1,104 @@
+{
+  cmake,
+  opencv,
+  udev,
+  libjpeg,
+  libGL,
+  fontconfig,
+  xorg,
+  opencvsharp,
+  lib,
+  buildDotnetModule,
+  dotnetCorePackages,
+  fetchFromGitHub,
+}:
+let
+  internal = builtins.fetchurl {
```

You should use `pkgs.fetchurl` instead, to avoid the pure eval error we can see in the CI. Make sure to add `fetchurl` to the args list above

```suggestion
  internal = fetchurl {
```